gamemodes/terrortown/gamemode/shared/hud_elements/tttwswitch/pure_skin_wswitch.lua

```diff
@@ -117,7 +99,9 @@ if CLIENT then
 	}
 
 	function HUDELEMENT:RecalculateBasePos()
-		self:SetBasePos(ScrW() - (width + self.margin * 2), ScrH() - self.margin)
+		local w = self:GetSize().w
+		self.margin = 5 * self.scale
```

Comment:
just as an example: Here, an empty line would be great :D Because the sense of this code differs in every line: At first, you've defined a local var, then you've updated an already existing var and at next you've called a function. Personally, I prefer code blocks with the same sense.